torchqdynamics/sesolve/exponentiate.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+from ..progress_bar import tqdm
+from ..qsolver import QSolver
+
+
+class SEExponentiate(QSolver):
```

Comment:
I think a better name would be `SEPropagator`. It is more explicit on what the solver is actually doing.